eng/SourceBuildPrebuiltBaseline.xml

```diff
@@ -4,14 +4,16 @@
 <UsageData>
   <IgnorePatterns>
     <UsagePattern IdentityGlob="Microsoft.SourceBuild.Intermediate.*/*" />
-
+    <UsagePattern IdentityGlob="System.Formats.Asn1/*8.0.1*" />
+    <UsagePattern IdentityGlob="System.Collections.Immutable/*8.0.0*" />
```

Comment:
I'm a bit confused as to why this is necessary. Doesn't this branch flow into .NET 8?

Unless msbuild is not targeting net8?

Reply:
VS 17.8 corresponds to 8.0.1xx, so it should, but I saw this error when I updated `S.C.Immutable`: https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=10348135&view=logs&j=2f0d093c-1064-5c86-fc5b-b7b1eca8e66a&t=52d0a7a6-39c9-5fa2-86e8-78f84e98a3a2&l=110

I guess @GangWang01 had same experience with `S.F.Asn1`.

Reply:
Okay - I think as long msbuild is not targeting net8 yet, these prebuilts make sense. The 7.0 versions weren't prebuilts because they're in SBRP, which also makes sense.

So as these are in the Version.Details.xml and each has a version prop, they should get lifted in the VMR to the latest version.